ui/app/templates/vault/cluster/clients/index.hbs

```diff
@@ -62,5 +63,6 @@
   </Toolbar>
   <Clients::Config @model={{this.model.config}} @isLoading={{this.currentlyLoading}} />
 {{else}}
```

Comment:
It might be cleaner to check for the tab here and render a different component rather than keeping it all in the Clients::Dashboard component. I know that the layouts are very similar, but that component is getting hard to follow and I'd prefer a little duplication over not being able to parse what the template should look like in a given state.

Reply:
Good idea. I'll put a comment in the code as right now changing this will break the layout for client count view 1.9.